cmd/run.go

```diff
@@ -78,5 +81,7 @@ func runCommandWithViperArgs(_ *cobra.Command, testTargets []string) error {
 		os.Stderr,
 		options,
 	)
-	return runner.RunCommand()
+	if err := runner.RunCommand(); err != nil {
+		os.Exit(1)
```

Comment:
Should we print this error ?

Reply:
Not really, as it would be sort of a duplication of the summary for errors.

Reply:
Ok